docs/kubernetes/user-guides/basic.md

```diff
@@ -46,18 +46,20 @@ web-server                1/1       Running   0          1m
 **Note:** Regional cluster support only available in beta starting with
```

Comment:
Can we add a note for GA? I guess this is aligning with the CSI topology feature.